### PR TITLE
Make html-differences.whatwg.org redirect

### LINKED ISSUE #100: Make html-differences.whatwg.org redirect to https://html.spec.whatwg.org/dev/
Once we've done that:

1. Close https://github.com/whatwg/meta/issues/2.
2. Archive https://github.com/whatwg/html-differences.

cc @zcorpan @sideshowbarker

### PR DESCRIPTION
**Pull request by annevk**
Fixes #100.

Note that I have not tried to deploy this, but I think this is what's needed based on looking at other files.

### CODE REVIEW

File: debian/marquee/README.md [hunk 1]
```diff
@@ -3,6 +3,5 @@ marquee hosts simple static sites.
 Repos that deploy to this server:
 - https://github.com/whatwg/books/blob/master/.travis.yml
 - https://github.com/whatwg/figures/blob/master/.travis.yml
-- https://github.com/whatwg/html-differences/blob/master/.travis.yml
```
Comment (foolip) [inline debian/marquee/README.md]: Shall delete that config and archive https://github.com/whatwg/html-differences to make sure that it's never deployed again?

Comment (annevk) [reply debian/marquee/README.md]: Yeah, that's the plan per #100.

**Comment by foolip** [comment]
@sideshowbarker can you deploy this?

**Comment by sideshowbarker** [comment]
deployed

**Comment by foolip** [comment]
Yay, thank you!

**Pull request merged**